RX20_16:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
  
  # Position Controllers ---------------------------------------
  tilt_joint_controller:
    type: effort_controllers/JointPositionController
    joint: tilt_joint
    pid: {p: 200.0, i: 0.5, d: 1.0}

  lift_joint_controller:
    type: effort_controllers/JointPositionController
    joint: lift_joint
    pid: {p: 200.0, i: 0.5, d: 1.0}

  sideshift_joint_controller:
    type: effort_controllers/JointPositionController
    joint: sideshift_joint
    pid: {p: 200.0, i: 0.5, d: 1.0}

  # wheel_left_steering_joint_controller:
  #   type: effort_controllers/JointPositionController
  #   joint: wheel_left_steering_joint
  #   pid: {p: 200.0, i: 0.0, d: 1.0}

  # wheel_right_steering_joint_controller:
  #   type: effort_controllers/JointPositionController
  #   joint: wheel_right_steering_joint
  #   pid: {p: 200.0, i: 0.0, d: 1.0}

  # wheel_left_driving_joint_controller:
  #   type: effort_controllers/JointVelocityController
  #   joint: wheel_left_driving_joint
  #   pid:
  #     p: 20.0
  #     i: 0.0
  #     d: 0.0
  #     i_clamp: 1000.0

  # wheel_right_driving_joint_controller:
  #   type: effort_controllers/JointVelocityController
  #   joint: wheel_right_driving_joint
  #   pid:
  #     p: 20.0
  #     i: 0.0
  #     d: 0.0
  #     i_clamp: 1000.0